lib/pbench/cli/agent/commands/results/move.py

```diff
@@ -0,0 +1,257 @@
+import os
+import shutil
+import socket
+import tempfile
+
+import click
+
+from pbench.agent.base import BaseCommand
+from pbench.agent.results import MakeResultTb, CopyResultTb
+from pbench.cli.agent import pass_cli_context, CliContext
+from pbench.cli.agent.options import common_options
+from pbench.common.exceptions import BadMDLogFormat
+from pbench.common.utils import validate_hostname
+
+
+class MoveResults(BaseCommand):
+    """MoveResutls - command implementation for "pbench results move."
```

Comment:
"MoveResults" ... but I think we probably shouldn't be repeating the method name; just another "double truth" that can get out of sync later with the code.